source/config.js

```diff
@@ -1,5 +1,5 @@
 export default {
-    API_CLIENT_ID: 'ae51d45d93313f1',
+    DEFAULT_API_CLIENT_ID: 'ae51d45d93313f1',
```

Comment:
@ReallyLiri Perhaps we can move the code that reads the users' custom client ID inside this file (and fallback to the default value if none is provided) and assign it to the `API_CLIENT_ID` key, so that in `source/axios.js` we can use `config.API_CLIENT_ID` directly without necessarily knowing the logic of where it came from. What do you think? 😄